### ISSUE TITLE
Backward compatibility for API <26

### ISSUE THREAD

**Issue by kamal-lab** [rafi0101/Android-Room-Database-Backup#2]
This library requires min API 26. It will be good if we get support for at least until API 21

**Comment by rafi0101** [comment]
@kamal-lab may you have a look at [feature/api21](https://github.com/rafi0101/Android-Room-Database-Backup/tree/feature/api21), test it with your application and give me some feedback, so I can merge it. I tested it already but only with an emulator.

**Comment by kamal-lab** [comment]
Thanks for the update. I didn't use your library because at the time I was supposed to implement backup feature immediately. So I understood your concept and made a java helper class(since I'm not good with kotlin yet).

I used like this,
`File backupFile = ...`
`File dbFile =...`
`FileChannel source = new FileInputStream(dbFile).getChannel();`
`FileChannel dest = new FileOutputStream(backupFile).getChannel();`
`dest.transferFrom(source, 0, source.size());`

And I checked that [feature/api21](https://github.com/rafi0101/Android-Room-Database-Backup/tree/feature/api21) which is also working fine with minSdkVersion 21 and I tested it on API 28 device. But I don't have API 21 device with me currently if it is necessary for you to confirm.
